spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/DefaultServerIntrospector.java

```diff
@@ -30,8 +30,12 @@
  */
 public class DefaultServerIntrospector implements ServerIntrospector {
 
-	@Autowired
-	private ServerIntrospectorProperties serverIntrospectorProperties;
+	private ServerIntrospectorProperties serverIntrospectorProperties = new ServerIntrospectorProperties();
+
+	@Autowired(required = false)
```

Comment:
I think this change was unneeded and should be reverted. The configuration properties bean is created by the auto-configuration.